### ISSUE TITLE
Support Windows?

### ISSUE THREAD

**Issue by TheThreeDog** [gravmatt/py-term#7]
I see that VT100 control code is used here, which can only be used under Linux shell. Have you considered how to support windows?

**Comment by gravmatt** [comment]
No, there are not plans. (I'm not a windows user and would not be able to test it.) 
But supporting windows terminals would be nice!

I found this article where it should be possible to enable VT100 Terminal codes for windows.

https://stackoverflow.com/questions/51680709/colored-text-output-in-powershell-console-using-ansi-vt100-codes
